docs/cloud/satellites/managing.md

```diff
@@ -146,6 +146,20 @@ Note that satellites on pinned versions may still receive auto-updates during a
 earthly satellite launch --version v0.6.29 my-satellite
 ```
 
+#### Weekends-Only Mode
+
+In cases where nightly maintenance schedules are not suitable, you can configure your auto-updates so that they will only run on Saturday using the following launch flag:
+
+```bash
+earthly satellite launch --maintenance-weekends-only my-satellite
+```
+
+{% hint style='info' %}
+##### Note
+At the time of writing, the day of the week is determined based on the UTC timezone.
```

Comment:
We should update this when behavior changes, rather than leave the user wondering if it's different now or not

```suggestion
The day of the week is determined based on the UTC timezone.
```